Esquilemos al zorro

Silverexperience opened the pull request:
## What Does This PR Do
Le quita el sentido del dolor, les devuelve su lentitud, sube su resistencia al daño bruto, su vida y  resistencia al daño de stamina pero aun reciben mas daño por quemaduras y por objetos filosos.
Las ninfas diona ahora necesitan oxigeno,

## Why It's Good For The Game
A nadie le gustan las diona rapidas y este las devuelve a como gustaban.
## Images of changes
### `Puñetazo humano`
![imagen](https://user-images.githubusercontent.com/58746682/74677474-b1615600-51b8-11ea-9d0c-4831dc43c031.png)
### `Puñetazo diona`
![imagen](https://user-images.githubusercontent.com/58746682/74677483-b8886400-51b8-11ea-8d9d-ef8e1ebc6251.png)
## Changelog
:cl:
add: Las ninfa diona necesitan oxigeno
balance: Las dionas resisten más daño bruto
balance: Las dionas no sienten dolor.
balance: Las dionas van mas lento.
balance: Las dionas son mas fuertes.
balance: Las dionas resisten mas daño de stamina.
balance: Las dionas son mas vulnerables a los objetos afilados.
balance: Las dionas tienen 120 de vida.
/:cl:

Silverexperience committed:
Vuelve a los "antiguos" diona

Silverexperience committed:
Quita la resistencia al espacio y el no respirar,tambien devuelve la sangre y la posiblida de romperse huesos

Silverexperience committed:
Un pequeño cambio

Silverexperience committed:
Otro cambio pequeño mas

Silverexperience committed:
Update diona.dm

Evankhell561 commented on code/modules/surgery/organs/subtypes/diona.dm:
```diff
@@ -102,4 +97,9 @@
 	icon_state = "claw"
-	alcohol_intensity = 0.5
\ No newline at end of file
+	alcohol_intensity = 0.5
```
devuelve esto justo a la linea de donde la sacaste.

Silverexperience replied:
> 
> 
> devuelve esto justo a la linea de donde la sacaste.

Devuelto, creo.

Silverexperience committed:
Ahora las ninfa diona necesitan oxigeno

Silverexperience committed:
Cambia la linea

Silverexperience committed:
Merge branch 'Antiguos-dionas' of https://github.com/Silverexperience/Paradise into Antiguos-dionas

Silverexperience committed:
Update diona.dm

Silverexperience committed:
Update diona.dm

Silverexperience committed:
Update diona.dm

Silverexperience committed:
Update diona.dm

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,11 +8,14 @@
 	heatmod = 1.5
 	var/pod = FALSE //did they come from a pod? If so, they're stronger than normal Diona.
 
+
```
quita esta linea inecesaria.

Silverexperience replied:
Quitado

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,11 +8,14 @@
 	remains_type = /obj/effect/decal/cleanable/ash
+	speed_mod = 5
 
+	brute_mod = 0.40
```
ponlo a 0.75

Silverexperience replied:
Puesto

Evankhell561 commented on code/modules/mob/living/simple_animal/friendly/diona.dm:
```diff
@@ -13,8 +13,7 @@
 	mob_size = MOB_SIZE_SMALL
 	ventcrawler = 2
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
```
que hace la variable minbodytemp = 0?

Silverexperience replied:
La temperatura minima que puede alcanzar el cuerpo de manera estable

Silverexperience committed:
Quita la linea y baja la resistencia

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,7 +8,9 @@
 	speech_chance = 20
 	unarmed_type = /datum/unarmed_attack/diona
 	remains_type = /obj/effect/decal/cleanable/ash
+	speed_mod = 5
```
si todo lo que vas a hacer es darle resistencia al daño fisico y que no sienta dolor no es necesario este nerf a la velocidad. si quieres nerfear esta caracteristica puedes subirla a 2.

Silverexperience replied:
Le subo el daño de los puñetazos tambien

Evankhell561 commented:
tambien arregla el changelog
balance: las dionas resisten más daño bruto
balance: las dionas no sienten dolor.

Silverexperience committed:
Les aumenta la fuerza de los puñetazos

Evankhell561 commented:
bien, entonces el concepto de diona mecanicamente es : un tanke de daño fisico que te puede reventar la cara a puñetazos.

Silverexperience commented:
> 
> 
> bien, entonces el concepto de diona mecanicamente es : un tanke de daño fisico que te puede reventar la cara a puñetazos.

Si, basicamente eso.

Evankhell561 commented:
me sigue pareciendo que cindo de velocidad es demasiado lento incluso para esto. y que subirlo a 4 no está mal.

Silverexperience committed:
Le sube la velocidad a 4

Silverexperience commented:
> 
> 
> me sigue pareciendo que cindo de velocidad es demasiado lento incluso para esto. y que subirlo a 4 no está mal.

Bien, lo subo a 4.

Evankhell561 commented:
yo le añadiria tambien resitencia a los disambler. o a los daños a la stamia. tambien le subiria un poco la vida. tal vez a 120 y le aumentaria el daño que recibien del fuego para compensar.

Evankhell561 commented:
También, se me ocurre podrian regenerar brazos y piernas.  como los slimes.

Evankhell561 commented:
incluso le añadiria que reciban más daño de cosas filosas.

Silverexperience commented:
> 
> 
> También, se me ocurre podrian regenerar brazos y piernas. como los slimes.

¿Y a que costo seria la regeneracion?, si no estaria bastante roto.

Evankhell561 commented:
> ¿Y a que costo seria la regeneracion?, si no estaria bastante roto.

cierto, olvide de que ya se curaban las heridas en la luz. olvidalo.

Silverexperience commented:
> 
> 
> > ¿Y a que costo seria la regeneracion?, si no estaria bastante roto.
> 
> cierto, olvide de que ya se curaban las heridas en la luz. olvidalo.

Lo demas podria intentar añadirlo

Evankhell561 commented:
> Lo demas podria intentar añadirlo

bien, tampoco digo que le des inmunidad al daño de stamia o a los disambler. solo que sea dificil tumbarlos de esa forma. debido a que ya son lentos de por sí.

Silverexperience committed:
Unos balanceos mas

Le da mas salud, le da mas resistencia a la stamina pero aumenta el daño por calor y los objetos filosos le hacen mas daño.

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -117,4 +123,8 @@
+
+/datum/species/diona/apply_damage(obj/item/W)
+	if(is_sharp(W))
+		brute_mod = W.force / 10
```
entre 5.

Silverexperience replied:
Eso quizas seria demasiado porque para objetos con 32 de daño afilados quedaria con 6,4 de debilidad al daño bruto.
Entre diez esos 32 quedarian en 3,2 de resistencia al daño bruto

Evankhell561 replied:
tienes 20 de vida extra man.

Silverexperience replied:
Bueno, lo dejare entre 5

Silverexperience committed:
Sube un poco el daño por armas afiladas

Silverexperience committed:
Merge remote-tracking branch 'upstream/master' into Antiguos-dionas

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,9 +8,15 @@
+	total_health = 120
+	speed_mod = 4
+	punchdamagelow = 7
+	punchdamagehigh = 14
```
esto no parece estar funcionado. no pega duro.

Silverexperience replied:
Ahora si pega mas duro o eso comprobe

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,9 +8,15 @@
+	punchdamagehigh = 14
+
+	brute_mod = 0.75
+	stamina_mod = 0.65
```
pon esto a 0.75

Silverexperience replied:
Listo.

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -8,9 +8,15 @@
+
+	brute_mod = 0.75
+	stamina_mod = 0.65
+	burn_mod = 1.3
```
segun el discord, le estamos dando mucho powa. subele esto a 1.4.

Silverexperience replied:
Listo.

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -117,4 +123,8 @@
+
+/datum/species/diona/apply_damage(obj/item/W)
+	if(is_sharp(W))
+		brute_mod = W.force / 5
```
las armas filosas tambien hacen daño filoso por lo que si le reduces el 25% y luego le aumentas el 25% es como si no le aumentaras nada. pon force / 2.

Silverexperience replied:
Listo

Silverexperience committed:
Unos cambios

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -117,4 +123,8 @@
+
+/datum/species/diona/apply_damage(obj/item/W)
+	if(is_sharp(W))
+		brute_mod = W.force / 2
```
solo para asegurarnos, al final de este proc pon: ..()

Silverexperience replied:
Listo

Silverexperience committed:
Un añadido

Silverexperience committed:
Puñetazos Fuertes

Evankhell561 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -10,8 +10,8 @@
-	punchdamagelow = 7
-	punchdamagehigh = 14
+	punchdamagelow = 17
+	punchdamagehigh = 19
```
no creo que solo aumentar el valor lo solucione... y esos valores si funcionan son demasiado altos. prueba poner 10 y 15 y testea.

Silverexperience replied:
Tengo imagenes de lo testeado en la descripcion del PR aunque si hace bastante daño.

Evankhell561 replied:
caules son estos valores para un humano?

Silverexperience replied:
De 0 a 9
![imagen](https://user-images.githubusercontent.com/58746682/74678485-6e54b200-51bb-11ea-9ae0-23ab558d884f.png)

Silverexperience committed:
Baja el daño
